content/posts/2021/launching-reana-blog.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "REANA blog is launched!"
+date: 2021-01-11T16:35:46+01:00
+tags:
+  - reana
+---
+
+We are glad to announce the launch of a new blog series about [REANA
+reproducible analysis platform](https://www.reana.io). We plan to
+describe new REANA releases, share exciting features and other useful
+tips and tricks for running containerised data analyses on REANA.
+
+You can subscribe to the [REANA blog feed](/index.xml) or get in touch at
+the [REANA user forum](https://forum.reana.io).
```

Comment:
In the rendered version REANA appears in every line, perhaps we can use it less and still keep it comprehensible?

```suggestion
We are glad to announce the launch of a new blog series about [REANA
reproducible analysis platform](https://www.reana.io). We plan to
describe new releases, share exciting features and other useful
tips and tricks for running containerised data analyses.

You can subscribe to the [REANA blog feed](/index.xml) or get in touch at
the [REANA user forum](https://forum.reana.io).
```